Maple2.Database/Model/CharacterUnlock.cs

```diff
@@ -3,6 +3,7 @@
 using System.Linq;
 using Maple2.Database.Extensions;
 using Maple2.Model.Enum;
+using Maple2.Model.Game;
```

Comment:
Can we create a separate Database internal FishEntry type so we don't need Game import here. I try to keep a layer of separation between the two, hence all the conversion logic.